docs/command.rst

```diff
@@ -0,0 +1,179 @@
+.. _command:
+
+Management Command
+==================
+
+Use ``python manage.py pglock`` to view, filter, and kill locks.
+
+Basic Usage
+-----------
+
+Running ``python manage.py pglock`` will show a list of active locks.
+Fields are separated by ``|`` and are configured with
+``settings.PGLOCK_ATTRIBUTES``, which defaults to the following:
+
+1. **activity_id**: The process ID that's using the lock.
+2. **wait_duration**: The time the lock has been waiting, or ``None`` if it's
+   acquired.
+3. **mode**: The mode of the lock, such as "ACCESS_EXCLUSIVE".
+4. **rel_kind**: The kind of relationship, such as "TABLE".
+5. **rel_name**: The name of the relationship locked, such as the table name.
+6. **activity__context**: Application context tracked by
+   `django-pgactivity <https://django-pgactivity.readthedocs.io>`__.
+7. **activity__query**: The SQL of the query.
+
+Output looks like the following::
+
+    76362 | 21:02:35 | ACCESS_EXCLUSIVE | TABLE | auth_user | None | lock table auth_user in access
```

Comment:
Does the output contain the column names?

Reply:
`-e` for "expanded" mode shows the full SQL and the column names.